portfolio.js

```diff
@@ -23,3 +23,28 @@ projectInfo.forEach(function(currentProject) {
 projects.forEach(function(project) {
   $('#projects').append(project.toHtml());
 });
+
+Project.loadAll() = function(projects) {
+  projects.forEach(function(ele) {
+    projects.push(new Project(ele));
+  });
+};
+
+Project.fetchAll() = function() {
```

Comment:
Ditto from above. You are calling this function in your definition. This will break your code!